modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/DumpCore_conf.pm

```diff
@@ -432,7 +433,7 @@ sub pipeline_analyses {
        },
       -can_be_empty    => 1,
       -max_retry_count => 1,
-      -hive_capacity   => 10,
+      -hive_capacity   => 20,
```

Comment:
Because hive_capacity spreads the load across many different dumping modules - using analysis_capacity instead would enable each module to operate independently, tying up all the database connections.

Reply:
:+1: